ui-v2/app/utils/computed/purify.js

```diff
@@ -39,4 +37,6 @@ export default function(computed, filter) {
     // concat/push the user function back on
     return computed(...args.concat([cb]));
   };
-}
+};
+export const subscribe = purify(computed);
+export default purify;
```

Comment:
The main changes here are to give us an easily preconfigured `subscribe` using `purify` as we don't want to have to set it up everytime we want to use it.